docs/source/quickstart.rst

```diff
@@ -85,18 +85,45 @@ Where the contents of ``payload.txt`` is:
 
 Walking through this specification:
 
-- The application name is specified as ``hello_world``, and the will be
-  deployed in the YARN queue ``default``.
+- The application name is specified as ``hello_world``, and will be deployed in
+  the YARN queue ``default``.
+
+  .. code-block:: yaml
+
+     name: hello_world
+     queue: default
 
 - The application starts a single service ``my_service`` on a container using 1
   virtual core and 128 MB of memory.
 
+  .. code-block:: yaml
+
+     services:
+       my_service:
+         resources:
+           vcores: 1
+           memory: 128
+
+
 - The file ``payload.txt`` is distributed with the application, and is named
   ``payload.txt`` both locally and on the running container.
 
+  .. code-block:: yaml
+
+     files:
+       payload.txt: payload.txt
```

Comment:
I tried maintaining the indentation of `files` and `commands`, and found this isn't possible without a filler character (indentation is stripped until first non-whitespace character is found). I'm not sure if this looks better or not, but here's one thought:
<img width="367" alt="screen shot 2018-08-08 at 12 38 30 pm" src="https://user-images.githubusercontent.com/2783717/43854308-271b9798-9b08-11e8-9fbc-bb41f2510feb.png">

Reply:
Actually, I think I do like this better, if you wouldn't mind adding it.